includes/Maps_DisplayMapRenderer.php

```diff
@@ -225,28 +229,35 @@ public static function evilOpenLayersHack( $layers ) {
 				}
 			}
 		}
-
-		MapsMappingServices::getServiceInstance( 'openlayers' )->addLayerDependencies( self::getLayerDependencies( $layerNames ) );
-
 		return $layerDefs;
 	}
 
-	/**
-	 * FIXME
-	 * @see evilOpenLayersHack
-	 */
-	private static function getLayerDependencies( array $layerNames ) {
-		global $egMapsOLLayerDependencies, $egMapsOLAvailableLayers;
+	public static function getLayerDependencies( $service, $params ) {
```

Comment:
If this class is the only user of this method, it makes no sense for this to be public and static. Which makes me think it's probably used elsewhere, so the added param might break something.